### PR TITLE
Updates clown smiling at me

### PR DESCRIPTION
**Pull request by Coxswain-Navigator**
## About The Pull Request
removes stun and updates doc with new info

## Why It's Good For The Game
Stuns are bad, abnormalities with no decent management system are worse.

## Changelog
:cl:
tweak: removes stun from clown smiling at me
balance: rebalanced clown's breach conditions
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/abnormality/waw/clown_smiling.dm [hunk 1]
```diff
@@ -116,7 +116,7 @@
 
 // Prevents knife throwing in mele range
 /mob/living/simple_animal/hostile/abnormality/clown/OpenFire(atom/A)
-	if(A in livinginrange(1,src))
+	if(A in livinginrange(2,src)) //no shooty in mele
```
Comment (EgorDinamit) [inline code/modules/mob/living/simple_animal/abnormality/waw/clown_smiling.dm:119]:
```suggestion
	if(get_dist(src, A) <= 2) //no shooty in mele
```
Never use livinginrange() or livinginview() for short(below 30) distances.

File: code/modules/paperwork/records/info/waw.dm [hunk 2]
```diff
@@ -501,8 +501,9 @@
 	Qliphoth Counter : 1	<br>
 	Work Damage Type : White	<br>
 	Work Damage : High	<br>
-	- When the work result was Normal, the Qliphoth Counter lowered with a normal probability.	<br>
+	- When the work result was Bad, the Qliphoth Counter lowered with a high probability.	<br>
```
Comment (EgorDinamit) [inline code/modules/paperwork/records/info/waw.dm:504]: Remove probability message.

File: code/modules/paperwork/records/info/waw.dm [hunk 3]
```diff
@@ -501,8 +501,9 @@
...
 	- When the work result was Good, the Qliphoth Counter lowered with a high probability.	<br>
+	- WARNING: Upon O-01-17's death, immediately evacuate the area surrounding O-01-17's corpse.	<br>
```
Comment (EgorDinamit) [inline code/modules/paperwork/records/info/waw.dm:506]: This sounds like it sets off a nuke. Just say that it deals red damage around it on death.

File: code/modules/projectiles/projectile/magic/abnormality.dm [hunk 4]
```diff
@@ -111,8 +111,19 @@
...
 /obj/projectile/clown_throw/Initialize()
 	. = ..()
 	SpinAnimation()
+
+/obj/projectile/clown_throw/on_hit(target)
+	if(!ishuman(target))
+		return ..()
+	var/mob/living/carbon/human/H = target
+	H.add_movespeed_modifier(/datum/movespeed_modifier/clowned)
+	addtimer(CALLBACK(H, /mob.proc/remove_movespeed_modifier, /datum/movespeed_modifier/clowned), 2 SECONDS, TIMER_UNIQUE | TIMER_OVERRIDE)
```
Comment (EgorDinamit) [inline code/modules/projectiles/projectile/magic/abnormality.dm:124]:
```suggestion
	addtimer(CALLBACK(H, .mob/proc/remove_movespeed_modifier, /datum/movespeed_modifier/clowned), 2 SECONDS, TIMER_UNIQUE | TIMER_OVERRIDE)
```
I think that's how you do it?

**Review by EgorDinamit — APPROVED** [review]

**Comment by Coxswain-Navigator** [comment]
All requested changes made, has been tested locally and everything works as intended.

**Pull request merged**